app/ide-desktop/lib/dashboard/README.md

```diff
@@ -31,3 +31,39 @@ management, project sharing, and user accounts and authentication.
   certain static hosting providers.
 - `esbuild-config.ts`: Configuration for ESBuild based on the environment
   variables. This is a dependency of `esbuild-config.ts` in sibling modules.
+
+## Cloud environment variables
+
+These are environment variables related to the cloud backend. If these variables
+are not set, the build will still work, however access to the cloud backend will
+be disabled.
+
+Note that `ENSO_CLOUD_ENV` may be set to instead load the files from a `.env`
+file. If `ENSO_CLOUD_ENV` is not set, or it is `production`, then variables are
+attempted to be read from `.env`. If it is set to any other value (say, `foo`),
+then it is loaded from `.foo.env`.
+
+(While the convention in the Node.js ecosystem is to name the variants like
+`.env.foo`, `.foo.env` has been chosen here because `.env` should be more like
+a file extension. Visual Studio Code also understands `.foo.env` but not
+`.env.foo`.)
+
+- `ENSO_CLOUD_REDIRECT`: The domain (or `localhost:8080`) where the login link
+  should redirect. Should include neither a path, nor a trailing slash.
+- `ENSO_CLOUD_ENVIRONMENT`: The name of backend environment matching the
+  provided configuration keys. For most builds this should be `production`,
+  meaning that requests go to the production cloud backend.
+- `ENSO_CLOUD_API_URL`: The root path for all API endpoints. Should not include
+  a trailing slash.
+- `ENSO_CLOUD_SENTRY_DSN`: The Sentry Data Source Name (DSN) for this
+  environment. This should normally be the same for all environments.
+- `ENSO_CLOUD_STRIPE_KEY`: Stripe's publishable client-side key.
+- `ENSO_CLOUD_CHAT_URL`: The URL for the WebSocket server serving as the chat
+  backend.
+- `ENSO_CLOUD_AMPLIFY_USER_POOL_ID`: The ID of the Amplify user pool.
```

Comment:
FWIW it's a "Cognito user pool" -- Amplify is the name of AWS' frontend SDK for interacting with Cognito. In other parts of the code (i.e., the backend) we refer to it as the Cognito user pool (since the backend doesn't use Amplify).

That being said this naming scheme was my mistake initially and it's probably not worth it to change it since the meaning is still clear.

Reply:
should be simple enough to change though